### PR TITLE
Change the content on Blockchain Developer Path

### PR DESCRIPTION
**Pull request by pranavvdesai**
## Related Issue 

- Changed the content on developer path
Closes: #46 

## Changes

- fixed the pakage.json issue

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

**Comment by shuklaritvik06** [comment]
Can you add some ss?

**Review by KukretiShubham — COMMENTED** [review]
Please show the ss. Now, we have whole card hyperlinked. Would be great, if we have only 'Take me There' button clickable, removed development button. Even this is okay. We can create a separate Issue for that. For now Share the SS.

**Comment by shuklaritvik06** [comment]
> Please show the ss. Now, we have whole card hyperlinked. Would be great, if we have only 'Take me There' button clickable, removed development button. Even this is okay. We can create a separate Issue for that. For now Share the SS.

Great suggestion shubham! We can create separate issue for that

**Comment by pranavvdesai** [comment]
> Please add some ss

yes, only the Take me there button is clickable

image

**Comment by shuklaritvik06** [comment]
Hey @pranavvdesai  please resolve the merge conflicts

**Comment by pranavvdesai** [comment]
> Hey @pranavvdesai please resolve the merge conflicts

Okay will do

File: src/index.html [hunk 1]
```diff
@@ -154,20 +154,23 @@ <h4 id="name" class="text-xl font-semibold">Whimsical</h4>
...
+            <h4 id="name" class="text-xl font-semibold mb-2">Figment</h4>
+            <p id="caption" class="text-gray-800 mt-2 mb-2">The Web 3 education platform for developers, by developers. Learn the Web 3 basics with 101 Pathways.
+            </p>
+            <div id="label" class="flex text-sm gap-x-3 mt-2">
+              <a href="https://learn.figment.io/">
               <button
-                class="font-semibold bg-blue-100 text-blue-600 text-white rounded-md py-2 px-4 focus:outline-none">
-                Design
+                class="font-semibold bg-blue-100 text-blue-600 mx-auto lg:mx-0 text-white rounded-md py-2 px-4 focus:outline-none">
+                Take Me There!
```
Comment (Panquesito7) [inline src/index.html:171]:
```suggestion
                Take me there!
```

File: src/index.html [hunk 2]
```diff
@@ -177,19 +180,20 @@ <h4 id="name" class="text-xl font-semibold">MindMaster</h4>
...
+          <div id="body" class="flex flex-col ml-5">
+            <h4 id="name" class="text-xl font-semibold mb-2">CryptoZombies</h4>
+            <p id="caption" class="text-gray-800 mt-2">CryptoZombies is an interactive school that teaches you all things technical about blockchains.</p>
+            <div id="label" class="flex text-sm gap-x-3 mt-2">
+              <a href="https://cryptozombies.io/">
               <button
-                class="font-semibold bg-blue-100 text-blue-600 text-white rounded-md py-2 px-4 focus:outline-none">
-                Design
+                class="font-semibold bg-blue-100 text-blue-600 mx-auto lg:mx-0 text-white rounded-md py-2 px-4 focus:outline-none">
+                Take Me There!
```
Comment (Panquesito7) [inline src/index.html:194]:
```suggestion
                Take me there!
```

File: src/index.html [hunk 3]
```diff
@@ -131,20 +131,20 @@ <h4 id="name" class="text-xl font-semibold">Framer</h4>
...
+          <div id="body" class="flex flex-col ml-5">
+            <h4 id="name" class="text-xl font-semibold mb-2">Learn with Whiteboard</h4>
+            <p id="caption" class="text-gray-800 mt-2">The skills required for a blockchain developer which is one of the in-demand professions today.</p>
+            <div id="label" class="flex text-sm gap-x-3 mt-2 ">
+              <a href="https://www.youtube.com/watch?v=x9l3y0cdpuU">
               <button
-                class="font-semibold bg-blue-100 text-blue-600 text-white rounded-md py-2 px-4 focus:outline-none">
-                Design
+                class="font-semibold bg-blue-100 text-blue-600 mx-auto lg:mx-0 text-white rounded-md py-2 px-4 focus:outline-none">
+                Take Me There!
```
Comment (Panquesito7) [inline src/index.html:145]:
```suggestion
                Take me there!
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by vinzvinci — COMMENTED** [review]
Can you provide the latest screenshot? 

Please review this @mkubdev

**Comment by shuklaritvik06** [comment]
@pranavvdesai why you closed this PR?? You can update this only. It becomes difficult to maintain when you create multiple PR's for one change

**Pull request closed without merging**